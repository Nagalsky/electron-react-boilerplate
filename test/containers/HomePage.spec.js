import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import HomePage from '../../app/containers/Home';

Enzyme.configure({ adapter: new Adapter() });

describe('containers', () => {
  describe('Home', () => {
    it('should render home page', () => {
      const tree = renderer.create(<HomePage />).toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
