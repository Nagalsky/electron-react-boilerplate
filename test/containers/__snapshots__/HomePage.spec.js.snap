// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers Home should render home page 1`] = `
<div>
  <div
    className="css-12yvtzk-HomePage"
  >
    Electron-Mobx-Boilerplate
  </div>
</div>
`;
